Makefile

```diff
@@ -149,25 +149,19 @@ built_ui_shared_js_files=$(shared_jsx_files:%.jsx=$(BUILT)/ui/%.js)
 built_add_on_shared_js_files=$(shared_jsx_files:%.jsx=$(BUILT)/add-on/chrome/content/%.js)
 built_standalone_shared_js_files=$(shared_jsx_files:%.jsx=$(BUILT)/standalone/content/%.js)
 
-$(BUILT)/standalone/content/shared/js:
-	@mkdir -p $@
-
 # We can't use $(shared_jsx_files) here because % rules don't accept that.
-$(BUILT)/standalone/content/shared/js/%.js: shared/js/%.jsx $(BUILT)/standalone/content/shared/js
+$(BUILT)/standalone/content/shared/js/%.js: shared/js/%.jsx
+	mkdir -p $(@D)
 	$(BABEL) $< --out-file $@
 
-$(BUILT)/add-on/chrome/content/shared/js:
-	@mkdir -p $@
-
 # We can't use $(shared_jsx_files) here because % rules don't accept that.
-$(BUILT)/add-on/chrome/content/shared/js/%.js: shared/js/%.jsx $(BUILT)/add-on/chrome/content/shared/js
+$(BUILT)/add-on/chrome/content/shared/js/%.js: shared/js/%.jsx
+	mkdir -p $(@D)
 	$(BABEL) $< --out-file $@
 
-$(BUILT)/ui/shared/js:
-	@mkdir -p $@
-
 # We can't use $(shared_jsx_files) here because % rules don't accept that.
-$(BUILT)/ui/shared/js/%.js: shared/js/%.jsx $(BUILT)/ui/shared/js
+$(BUILT)/ui/shared/js/%.js: shared/js/%.jsx
+	@mkdir -p $(@D)
```

Comment:
This line has `@mkdir` while the rest are echo-y mkdir. I think it makes sense to suppress with @ as it could get spammy when building many jsx.